pom.xml

```diff
@@ -23,10 +26,11 @@
   </scm>
 
   <properties>
+    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
```

Comment:
don't we set it on parent pom?

Reply:
Yep, good point. Removed from `scalecube-organization-service` pom.